flocker/control/test/persisted_model.json

```diff
@@ -149,6 +152,10 @@
                     "flocker.control._model.Node"
                 ]
             },
+            "flocker.control._model.PersistentState": {
+                "category": "record",
+                "fields": {}
```

Comment:
:grey_question: Trying to review this without looking too far ahead in your PR chain... does this mean that we should expect fields to be added here, and that for every field that is added we rev the version of the persistence state?

I'm still not sure when I as a developer should add a field here without additional documentation.